spec/handlers/c/method_handler_spec.rb

```diff
@@ -226,6 +227,7 @@
     eof
     foo = Registry.at('Foo#foo?')
     expect(foo.tag(:return).types).to eq ['String']
+    expect(foo.tags(:return).size).to eq 1
```

Comment:
This is the test that catches the bug. I can dig deeper into why the other test didn't raise any errors before the fix.